### ISSUE TITLE
Negative tabindex attributes on vertical buttons removes them from the tab order

### ISSUE THREAD

**Issue by cryptasian** [microsoft/BotFramework-WebChat#4986]
### Is it an issue related to Adaptive Cards?

No.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### What is the public URL for the website?

https://edfinancial.com

### How to reproduce the issue?

This is feedback from the accessibility company Allyant who has been testing our website and has people who use screenreaders and other assistive technologies as a part of their testing team.

When tabbing through the vertical button options, tabbing takes you out of the button list options and to the input field. The button list can only be navigated with arrow keys. Each element should be focusable by the keyboard when navigating with the Tab and Shift + Tab keys.

![tabbing-vertical-buttons](https://github.com/microsoft/BotFramework-WebChat/assets/30207740/d145b044-b75d-4e34-a54e-e89bd9a5ad52)

### What do you expect?

Remove the tabindex attribute from each element so that they can be focused by the keyboard when navigating with the Tab and Shift + Tab keys. 

### What actually happened?

Currently, tabbing gets you to the button list, but then you have to use up/down arrows to navigate. Tabbing takes you away to the input field. 

### WCAG reference

https://www.w3.org/WAI/WCAG22/Understanding/keyboard

**Comment by compulim** [comment]
Suggested actions are toolbar pattern implementing the roving tabindex. We are following the APG of toolbar which suggests the roving tabindex, https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/.

To jump across actions, in a toolbar button, PWD will use <kbd>UP</kbd>/<kbd>DOWN</kbd> arrow keys. <kbd>TAB</kbd> will put the focus on the toolbar, and not individual buttons.
